src/Symfony/Component/DependencyInjection/Tests/Loader/XmlFileLoaderTest.php

```diff
@@ -207,6 +208,35 @@ public function testParsesTags()
         }
     }
 
+    /**
+     * @expectedException \InvalidArgumentException
+     */
+    public function testParseTagsWithoutNameThrowsException()
+    {
+        try {
+            $container = new ContainerBuilder();
+            $loader = new XmlFileLoader($container, new FileLocator(self::$fixturesPath.'/xml'));
+            $loader->load('badtag2.xml');
+        } catch (InvalidArgumentException $e) {
+            throw $e->getPrevious();
```

Comment:
this seems useless as the exception message is not tested against anyway. so you can just change `@expectedException` to the components one